pom.xml

```diff
@@ -127,8 +127,25 @@
             <optional>true</optional>
         </dependency>
         <dependency>
-            <groupId>ch.qos.logback</groupId>
-            <artifactId>logback-access</artifactId>
+            <groupId>ch.qos.logback.access</groupId>
+            <artifactId>common</artifactId>
+            <version>2.0.1</version>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback.access</groupId>
+            <artifactId>tomcat</artifactId>
+            <version>2.0.1</version>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback.access</groupId>
+            <artifactId>jetty12</artifactId>
+            <version>2.0.1</version>
+        </dependency>
```

Comment:
Is it possible to use the version of "ch.qos.logback.access" defined in spring-boot-dependencies (without specifying the version)?
Basically because I want to follow the versions defined in spring-boot-dependencies.

Reply:
logback-access v2

Reply:
Thanks.
(Regarding the version of "ch.qos.logback.access" defined in spring-boot-dependency, I understood that it was removed in Spring Boot 3.3.)